pyramid_storage/gcloud.py

```diff
@@ -50,14 +50,16 @@ def from_settings(cls, settings, prefix):
             ('gcloud.auto_create_bucket', False, False),
             ('gcloud.auto_create_acl', False, DEFAULT_BUCKET_ACL),
             ('gcloud.cache_control', False, None),
+            ('gcloud.uniform_bucket_level_access', False, False),
         )
         kwargs = utils.read_settings(settings, options, prefix)
         kwargs = dict([(k.replace('gcloud.', ''), v) for k, v in kwargs.items()])
         return cls(**kwargs)
 
     def __init__(self, credentials, bucket_name, project=None, acl=None, base_url='',
                  extensions='default', auto_create_bucket=False,
-                 auto_create_acl=DEFAULT_BUCKET_ACL, cache_control=None):
+                 auto_create_acl=DEFAULT_BUCKET_ACL, cache_control=None,
+                 uniform_bucket_level_access=False):
```

Comment:
Can we set `auto_create_acl`  to `None` here, and raise a `pyramid_exceptions.ConfigurationError` if `auto_create_acl` is set and `uniform_bucket_level_access` is `True`?